### ISSUE TITLE
no idea what to do getting this from Markbot

### ISSUE THREAD

**Issue by jenn-sweet** [jenn-sweet/kraken-good-accessibility#1]
like I said no idea getting these from Markbot

<img width="427" alt="Screen Shot 2021-10-06 at 12 42 31 PM" src="https://user-images.githubusercontent.com/77341921/136247309-c5bd4152-11ee-488a-afa8-841901f679cb.png">

@thatkidmadrid

**Comment by thatkidmadrid** [comment]
on line 17 of your html you have an opening tag`<div>` that doesn't have a closing tag`</div>`

**Comment by thatkidmadrid** [comment]
![Screen Shot 2021-10-06 at 12 55 40 PM](https://user-images.githubusercontent.com/6340290/136249249-4b12d297-cb6c-4a3d-892b-93daabcd66b1.png)

A good way to troubleshoot errors like this is to go into your text editor and use the collapse feature to collapse chunks of code that have proper open/close tags (the toggle icons appear beside the line number for collapsable elements)

In this screenshot I copied your code, and collapsed the header on line 11, a div on line 18, the main on line 45, etc. and this made the unclosed div on line 17 easier to find

**Comment by thatkidmadrid** [comment]
Also, the last error has to do with the leading 0 in `0.5rem` vs `.5rem`

In my opinion this isn't a big deal, but it's something markbot is looking out for.

I did some digging on this and it's essentially Google and other web leaders trying to standardize the way css is written. Also for large projects the leading 0s can add some extra heft to the code.

**Comment by jenn-sweet** [comment]
Thank you that fixed the issues I was having but
I can not get it to match the example and the more I do the more it looks less like it..so at this point I have no idea

**Comment by thatkidmadrid** [comment]
I've noticed you've made a few commits that are altering the css. We went over in class that there is very very  minimal css for this exercise. 95% of the styling is already built-in and will appear as you replace the generic markup in the index.html with the proper semantic html tags.

Without touching the css at all, and just replacing the tags, your website should match the screenshot almost exactly, and then you just need to style the focus/hover states of the navigation.

In the screenshot one of the buttons has keyboard focus and shows how it is styled.
